cypress/integration/dashboard/dashboard_spec.js

```diff
@@ -0,0 +1,74 @@
+function archiveAllDashboards() {
+  cy.visit('/dashboards');
+
+  // archive all dashboards
+  cy.getByTestId('DashboardLayoutContent').then($wrapper => {
+    $wrapper.find('.table-main-title').each((_, { href }) => {
+      archiveDashboard(href);
+    });
+  });
+}
```

Comment:
Personally I would not reset dashboards, neither before nor after. But if you want to guarantee a clean state here it will be better to use requests in this case:
```javascript
  cy.request('/api/dashboards').then((response) => {
    response.body.results.forEach((dashboard) => {
      cy.request('DELETE', `/api/dashboards/${dashboard.slug}`)
    });
  });
```

Reply:
Totally perfect.

But you're right, a better approach is just not to depend on clean state. I'll fix the code. 🙇

Reply:
OMG I'm in love with the code, now doesn't require clean slate:
1. When creating a new dashboard, listening to api response and extracting slug.
2. Slug then used as test hook.

💪

Reply:
> When creating a new dashboard, listening to api response and extracting slug.

That was really cool :ok_hand: